docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
 
-_The ability of a TRE operator to measure and control quality of processes, documentation and outputs._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-resource-allocation
 
-### Document management
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure there are adequate resources to provide assurance and meet information governance requirements.
+  - There should be adequate access to funding to implement and maintain the TRE which is under the direct control of the TRE organisation.
+  - Mandatory
+```
+
+## Quality Management
+
+What the organisation does to measure and control quality of processes, documentation and outputs.
+
+**Document and SOP Management:** Creating, organising, updating, and controlling documents and Standard Operating Procedures (SOPs) within the TRE organisation.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-document-management
+:name: tab-ig-quality-management
 
 * - Statement
   - Guidance
   - Importance
-* - You must control all of your policies and standard operating procedures.
-  - This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE operators.
+* - You must ensure all policies & standard operating procedures relevant to the TRE organisation are controlled.
+  - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.
   - Mandatory
-* - You should use codified change processes when altering your policies and standard operating procedures.
-  -
-  - Recommended
-* - You could use version control to track changes to their policies and processes.
+* - You must ensure all policies & standard operating procedures relevant to the TRE organisation are version controlled and have codified change processes.
   - Version control includes recording dates of changes, person responsible for carrying out changes, and summary of changes.
-  - Optional
+  - Mandatory
 ```
 
-### Issue management
+**Quality Management Reporting:** The generation and dissemination of reports or dashboards that provide insights and metrics on the performance and effectiveness of quality management processes and activities.
 
-_The ability of a TRE operator to track deviations from stated policies._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-quality-management-reporting
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must measure the performance of information governance within the TRE with regular reporting available to top management.
+  - This may include reports and dashboards showing security incidents, quality management deviations and audit findings.
+  - Mandatory
+```
+
+**Internal Audit:** An independent evaluation process within the TRE organisation that assesses and improves its internal controls, risk management, and governance.
```

Comment:
Based on https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#compliance-monitoring-and-reporting